fastly/errors.go

```diff
@@ -169,6 +169,18 @@ var ErrMissingCertBlob = errors.New("missing required field 'CertBlob'")
 // a "IntermediatesBlob" key, but one was not set.
 var ErrMissingIntermediatesBlob = errors.New("missing required field 'IntermediatesBlob'")
 
+// ErrMissingTLSCertificate is an error that is returned from an input struct that requires
+// a "TLSCertificate" struct, but one was not set.
+var ErrMissingTLSCertificate = errors.New("missing required field 'TLSCertificate'")
+
+// ErrMissingTLSConfiguration is an error that is returned from an input struct that requires
+// a "TLSConfiguration" struct, but one was not set.
```

Comment:
As I discuss above, a suggested change could be...

```suggestion
// a "Configuration" field assigned a "TLSConfiguration" struct, but one was not set.
```